scripts/pre-deploy-guard.sh

```
@@ -0,0 +1,129 @@
+    ok "B4 prod SSH" "ALPHA_VPS_IPV4 is set ($ALPHA_VPS_IPV4)"
+  elif [[ -n "${ALPHA_VPS_HOST:-}" && "${ALPHA_VPS_HOST:-}" != *":"* ]]; then
+    ok "B4 prod SSH" "no ALPHA_VPS_IPV4 but ALPHA_VPS_HOST is a usable IPv4 ($ALPHA_VPS_HOST)"
+  else
+    failed "B4 prod SSH" "prod env has only unreachable IPv6/host. export ALPHA_VPS_IPV4 (the reachable IPv4 from your deploy env) before the prod run."
+  fi
+else
+  ok "B4 prod SSH" "n/a (dev surface)"
+fi
+
+## --- B5 shared clone not on a foreign branch (parallel-session hijack) ---
+if [[ -f .git/worktrees/*/gitdir ]]; then
```

Shellcheck (0.11.0)

[error] -f doesn't work with globs. Use a for loop.

(SC2144)